packages/pg-meta/src/pg-meta-functions.ts

```diff
@@ -34,13 +58,11 @@ export const pgFunctionZod = z.object({
   behavior: z.union([z.literal('IMMUTABLE'), z.literal('STABLE'), z.literal('VOLATILE')]),
   security_definer: z.boolean(),
   config_params: z.union([z.record(z.string(), z.string()), z.null()]),
-})
+}) satisfies z.ZodType<PGFunction>
 
 export const pgFunctionArrayZod = z.array(pgFunctionZod)
 export const pgFunctionOptionalZod = z.optional(pgFunctionZod)
 
-export type PGFunction = z.infer<typeof pgFunctionZod>
-
```

Comment:
Pourquoi ?

Reply:
Good catch, that's a remaining artifact of me trying out some things to debug the types I had in my tests 😅 

Reverted this back to being infered by zod schema.